config_test.go

```diff
@@ -23,6 +23,9 @@ func TestConfig(t *testing.T) {
 	NewConfig().SetCraneliftOptLevel(OptLevelSpeed)
 	NewConfig().SetCraneliftOptLevel(OptLevelSpeedAndSize)
 	NewConfig().SetProfiler(ProfilingStrategyNone)
+	NewConfig().SetTarget("x86_64-unknown-linux-gnu")
```

Comment:
This technically only will work on x86_64 hosts since `all-arch` is frequently disabled in the C API (e.g. no cross-compilation), so could this be gated to only happen when the host target architecture is x86_64?